paket.dependencies

```diff
@@ -7,6 +7,7 @@ group Main
     nuget Microsoft.Extensions.Logging
     nuget FSharp.Core ~> 4.7
     nuget Taskbuilder.fs ~> 2.1
+nuget Thoth.Json.Net
```

Comment:
Make a separate group for `Examples` as we don't want `Thoth.Json.Net` or `Newtonsoft.Json` to be dependencies for the core library.